docs/src/changelog.md

```diff
@@ -18,6 +18,10 @@ If your currently installed *Quality-time* version is v4.0.0 or older, please re
 
 - When using SonarQube security hotspots as source for the security warnings metric, allow for filtering hotspots by hotspot status ('to review', 'acknowledged', 'fixed', 'safe'). Closes [#5956](https://github.com/ICTU/quality-time/issues/5956).
 
+### Changed
+
+- Write a more descriptive issue summary and description for issues created from Quality-time. Closes [#4747](https://github.com/ICTU/quality-time/issues/4747).
```

Comment:
Although the ticket lists adding an instruction to the manual, I think that https://quality-time.readthedocs.io/en/latest/usage.html#managing-technical-debt is already very descriptive. Listing more details in either changelog or docs will only make it more likely that they become outdated